apps/opik-backend/src/main/java/com/comet/opik/api/error/JsonInvalidFormatExceptionMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.comet.opik.api.error;
+
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.ExceptionMapper;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class JsonInvalidFormatExceptionMapper implements ExceptionMapper<InvalidFormatException> {
+
+    @Override
+    public Response toResponse(InvalidFormatException exception) {
+        log.warn("Deserialization exception: {}", exception.getMessage());
```

Comment:
Minor: this is caused by bad request, no need to log as warning level, probably info is fine. Only if many infos might be an indicator of something broken on our side. Otherwise, most likely users just making mistakes.